### PR TITLE
Content update 7 Feb

### PR DESCRIPTION
**Pull request by gkindavishal**

### CODE REVIEW

File: config/authors/amit-chandak-unskript.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "amit-chandak-unskript",
+  "bio": "",
```
Comment (5wathi) [inline config/authors/amit-chandak-unskript.json:3]: Co-Founder/Chief Architect - unSkript, Inc

File: config/authors/daria.json [hunk 2]
```diff
@@ -2,5 +2,6 @@
   "name": "Daria Orlova",
   "bio": "",
```
Comment (5wathi) [inline config/authors/daria.json:3]: Android & Flutter developer @ChiliLabs

File: config/authors/jayasimha-raghavan-unskript.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Jayasimha Raghavan",
+  "bio": "",
```
Comment (5wathi) [inline config/authors/jayasimha-raghavan-unskript.json:3]: Member Of Technical Staff, unSkript, Inc

File: config/authors/amit-chandak-unskript.json (hunk 1 above)
Comment (gkindavishal) [reply config/authors/amit-chandak-unskript.json]: Updated

File: config/authors/daria.json (hunk 2 above)
Comment (gkindavishal) [reply config/authors/daria.json]: Updated

File: config/authors/jayasimha-raghavan-unskript.json (hunk 3 above)
Comment (gkindavishal) [reply config/authors/jayasimha-raghavan-unskript.json]: Updated

File: config/authors/shloka-bhalgat-unskript.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "shloka-bhalgat-unskript",
+  "bio": "",
```
Comment (5wathi) [inline config/authors/shloka-bhalgat-unskript.json:3]: Senior Member of Technical Staff at unSkript, Inc

**Comment by 5wathi** [comment]
Left some comments on the bio of new authors @gkindavishal. LMK if you are able to view those comments. Thanks!

File: config/authors/shloka-bhalgat-unskript.json (hunk 4 above)
Comment (gkindavishal) [reply config/authors/shloka-bhalgat-unskript.json]: Updated @5wathi

**Comment by gkindavishal** [comment]
Hi @5wathi 
Unable to work on this branch due to the CLA issue with your account. Closing this PR.
New PR: https://github.com/google/devlibrary/pull/607

**Pull request closed without merging**